iphone/iphone/Titanium_Prefix.pch

```diff
@@ -17,6 +17,12 @@
 #define IS_SDK_IOS_14 false
 #endif
 
+#if __IPHONE_OS_VERSION_MAX_ALLOWED >= 130400
+#define IS_SDK_IOS_134 true
```

Comment:
We might want to change this macro  to  IS_SDK_IOS_13_4.  Convention should be IS_SDK_IOS_major_minor.